react/features/token-data/actions.js

```diff
@@ -14,17 +15,31 @@ import TokenData from './TokenData';
 export function parseTokenData() {
     return (dispatch: Dispatch<*>, getState: Function) => {
         const state = getState();
+        const { app } = state['features/app'];
+        const baseUrl = app._getDefaultURL();
```

Comment:
calling a private function in a different module doesn't sound like a good idea.